packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -112,127 +50,37 @@ window.\$hotReloadHook = function(modules) {
-      return el[0].getAttribute("href");
-    }
-  }
-  // return default value
-  return "/";
-}());
-$_currentDirectoryScript
-// dart loader
-if(!window.\$dartLoader) {
-   window.\$dartLoader = {
-     appDigests: _currentDirectory + 'basic.digests',
-     moduleIdToUrl: new Map(),
-     urlToModuleId: new Map(),
-     rootDirectories: new Array(),
-     // Used in package:build_runner/src/server/build_updates_client/hot_reload_client.dart
-     moduleParentsGraph: new Map(),
-     moduleLoadingErrorCallbacks: new Map(),
-     forceLoadModule: function (moduleName, callback, onError) {
-       if (typeof onError != 'undefined') {
-         var errorCallbacks = \$dartLoader.moduleLoadingErrorCallbacks;
-         if (!errorCallbacks.has(moduleName)) {
-           errorCallbacks.set(moduleName, new Set());
-         }
-         errorCallbacks.get(moduleName).add(onError);
-       }
-       requirejs.undef(moduleName);
-       requirejs([moduleName], function() {
-         if (typeof onError != 'undefined') {
-           errorCallbacks.get(moduleName).delete(onError);
-         }
-         if (typeof callback != 'undefined') {
-           callback();
-         }
-       });
-     },
-     getModuleLibraries: null, // set up by _initializeTools
-   };
-}
-let modulePaths = {};
-let customModulePaths = {};
-window.\$dartLoader.rootDirectories.push(window.location.origin + baseUrl);
-for (let moduleName of Object.getOwnPropertyNames(modulePaths)) {
-  let modulePath = modulePaths[moduleName];
-  if (modulePath != moduleName) {
-    customModulePaths[moduleName] = modulePath;
-  }
-  var src = window.location.origin + '/' + modulePath + '.js';
-  if (window.\$dartLoader.moduleIdToUrl.has(moduleName)) {
-    continue;
-  }
-  \$dartLoader.moduleIdToUrl.set(moduleName, src);
-  \$dartLoader.urlToModuleId.set(src, moduleName);
-}
 // Create the main module loaded below.
 define("main_module.bootstrap", ["$entrypoint", "dart_sdk"], function(app, dart_sdk) {
   dart_sdk.dart.setStartAsyncSynchronously(true);
   dart_sdk._debugger.registerDevtoolsFormatter();
-  \$dartLoader.getModuleLibraries = dart_sdk.dart.getModuleLibraries;
-  if (window.\$dartStackTraceUtility && !window.\$dartStackTraceUtility.ready) {
-    window.\$dartStackTraceUtility.ready = true;
-    let dart = dart_sdk.dart;
-    window.\$dartStackTraceUtility.setSourceMapProvider(
-      function(url) {
-        url = url.replace(baseUrl, '/');
-        var module = window.\$dartLoader.urlToModuleId.get(url);
-        if (!module) {
-          if (url.endsWith('dart_sdk.js')) {
-            module = 'dart_sdk';
-          } else {
-            try {
-              module = '/packages' + url.split('packages')[1].split('.')[0] + '.dart';
-            } catch (e) {
-              return null;
-            }
-          }
-        }
-        return dart.getSourceMap(module);
-      });
-  }
+  dart_sdk._isolate_helper.startRootIsolate(() => {}, []);
```

Comment:
I'm not familiar with this. What is this for?

Reply:
I think I copied this from the build_runner tooling, so its probably just a straggler. Will remove once I verify its not necessary

Reply:
Not necessary removed